url-shortner-view/src/App.js

```diff
@@ -5,13 +5,19 @@ import View from './View/View';
 import Footer from './Footer/Footer';
 import SecondPageView from './SecondPageView/SecondPageView';
 import { Route } from 'react-router-dom';
-
-function App() {
+import Spinner from './Spinner/Spinner';
+function App(props) {
   const [url, setUrl] = useState('');
   const [shortUrl, setShortUrl] = useState('');
+  const [spinner, setSpinner] = useState(false);
 
+  let spinnerDiv = '';
+  if (spinner) {
+    spinnerDiv = <Spinner />;
+  }
   return (
     <div className='App'>
+      {spinnerDiv}
```

Comment:
no need for spinnerDiv variable, use <Spinner /> directly. to show it google: react conditionally show a div